### PR TITLE
Update RxSwift to latest version (5.0)

### PR DESCRIPTION
**Pull request by farshadtx**

### CODE REVIEW

**Comment by farshadtx** [comment]
@freak4pc can you help on this?

File: RxSwiftUtilities.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -847,7 +873,7 @@
 				PRODUCT_NAME = RxSwiftUtilities;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```
Comment (freak4pc) [inline RxSwiftUtilities.xcodeproj/project.pbxproj]: No reason to raise the minimum to 5.0 as RxSwift 5 still supports Swift 4.2

File: Source/iOS/TwoWayBinding.swift [hunk 2]
```diff
@@ -38,7 +38,7 @@ private func nonMarkedText(_ textInput: UITextInput) -> String? {
     return (textInput.text(in: startRange) ?? "") + (textInput.text(in: endRange) ?? "")
 }
 
-public func <-> <Base>(textInput: TextInput<Base>, variable: Variable<String>) -> Disposable {
+public func <-> <Base>(textInput: TextInput<Base>, variable: BehaviorRelay<String>) -> Disposable {
```
Comment (freak4pc) [inline Source/iOS/TwoWayBinding.swift]:
```suggestion
public func <-> <Base>(textInput: TextInput<Base>, relay: BehaviorRelay<String>) -> Disposable {
```

File: Source/iOS/TwoWayBinding.swift [hunk 3]
```diff
@@ -61,7 +61,7 @@ public func <-> <Base>(textInput: TextInput<Base>, variable: Variable<String>) -
              and you hit "Done" button on keyboard.
              */
             if let nonMarkedTextValue = nonMarkedTextValue, nonMarkedTextValue != variable.value {
-                variable.value = nonMarkedTextValue
+                variable.accept(nonMarkedTextValue)
```
Comment (freak4pc) [inline Source/iOS/TwoWayBinding.swift]:
```suggestion
                relay.accept(nonMarkedTextValue)
```

**Review by freak4pc — CHANGES_REQUESTED** [review]
Looks good! One note and a few renames. Thanks!

File: RxSwiftUtilities.xcodeproj/project.pbxproj (hunk 1 above)
Comment (farshadtx) [reply RxSwiftUtilities.xcodeproj/project.pbxproj]: I think this changed automatically by XCode. I will revert it and check if it builds.

File: Source/iOS/TwoWayBinding.swift (hunk 2 above)
Comment (farshadtx) [reply Source/iOS/TwoWayBinding.swift]: @freak4pc renamed all instances! please review it again.

**Review by freak4pc — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**